vms/platformvm/block/executor/manager.go

```diff
@@ -124,6 +125,15 @@ func (m *manager) VerifyTx(tx *txs.Tx) error {
 		return ErrChainNotSynced
 	}
 
+	// If partial sync is enabled, this node isn't guaranteed to have the full
```

Comment:
>  If partial sync is enabled, this node isn't guaranteed to have the full UTXO set from shared memory

Can you explain why? I thought the partial sync flag just skips syncing every chain but the P-chain. But we're still only processing P-chain UTXOs in platformVM... no?

Reply:
oh nevermind I think I got it, we're importing UTXOs from other chains, is that it?